src/components/RadioButton/RadioButton.js

```diff
@@ -112,35 +113,42 @@ const RadioButtonLabel = styled('label')`
 /**
  * RadioButton component for forms.
  */
-const RadioButton = ({ onToggle, children, name, ...props }) => (
-  <Fragment>
-    <RadioButtonInput
-      id={name}
-      type="radio"
-      onClick={onToggle}
-      {...{ ...props, name }}
-    />
-    <RadioButtonLabel htmlFor={name} {...props}>
-      {children}
-    </RadioButtonLabel>
-  </Fragment>
-);
+const RadioButton = ({ onToggle, children, id, ...props }) => {
+  const defaultId = uniqueId('radio-button_');
```

Comment:
Inline this one 😄

Reply:
Can't see above comment. I'll move the OR operator out of the JSX though. :)